universe/index.js

```diff
@@ -0,0 +1,49 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of vikram.hello
+//
+// Copyright 2021 undefined <undefined>
+//
+// See LICENSE for details
+"use strict";
+const Tp = require("thingpedia");
+const querystring = require("querystring");
+const assert = require('assert');
+
+const tmdbAccess = "https://api.themoviedb.org/3/";
+const movieSearch = "search/movie?api_key=654083fa9e049ea234e4ae94d3e65774&language=en-US&query=";
+
+module.exports = class MovieClass extends Tp.BaseDevice {
+    //constructor taken from Yelp index.js
+    constructor(engine, state){
+        super(engine, state);
+        this.uniqueId = "vikram.hello";
+        this._queryResults = new Map();
+        this._deviceState = new Map ();
```

Comment:
What are these maps for? You don't seem to use them anywhere.

If you don't need them, the whole constructor becomes redundant because the default implementation will set uniqueId, name, and description already based on the manifest.